iotjs/iotjs-sample.ini

```diff
@@ -21,6 +21,7 @@ timeout=
 # Choose a command/env pair from below or define a new one here.
 build=${iotjs.build.debug_gcc_asan:command}
 build_env=${iotjs.build.debug_gcc_asan:env}
+validate_after_update=True
```

Comment:
These lines (with the `validate_after_update` settings) should go after the age lines, since they belong together.

Reply:
Done.